### ISSUE TITLE
Variant chunktable bad error message

### ISSUE THREAD

**Issue by hardingnj** [cggh/scikit-allel#109]
eg:

```
allele_depths = allel.VariantChunkedTable(fh[chrom]["calldata"], names=["AD"])

# ie forget to use 'AD'
allele_depths[:, 0]
```

> ---------------------------------------------------------------------------
> TypeError                                 Traceback (most recent call last)
> <ipython-input-43-12128e27c753> in <module>()
> ----> 1 allele_depths[:, 1]
> 
> /anaconda/envs/science/lib/python3.5/site-packages/allel/chunked/core.py in __getitem__(self, item)
>     821 
>     822         else:
> --> 823             raise NotImplementedError('item not suppored: %r' % item)
>     824 
>     825     def __array__(self):
> 
> TypeError: not all arguments converted during string formatting

Typo in 'supported'
Should throw a TypeError: unhashable type: 'slice'?

**Comment by alimanfoo** [comment]
Meant to say, thanks for this, error handling could indeed be better here.

**Comment by alimanfoo** [comment]
Resolved in #127.
